Packs/CommonScripts/Scripts/PcapHTTPExtractor/PcapHTTPExtractor.py

```diff
@@ -368,38 +376,48 @@ def get_markdown_output(http_flows):
+
+        if "allowedContentTypes" not in demisto.args():
+            ALLOWED_CONTENT_TYPES = ("text", "application/json", "multipart/form-data",
+                                     "application/xml", "application/xhtml+xml",
+                                     "application/ld+json", "application/javascript",
+                                     "multipart/alternative", "application/x-www-form-urlencoded")
+        else:
+            ALLOWED_CONTENT_TYPES = tuple(demisto.args()["allowedContentTypes"].split(","))  # type: ignore
 
-    if "allowedContentTypes" not in demisto.args():
-        ALLOWED_CONTENT_TYPES = ("text", "application/json", "multipart/form-data",
-                                 "application/xml", "application/xhtml+xml",
-                                 "application/ld+json", "application/javascript",
-                                 "multipart/alternative", "application/x-www-form-urlencoded")
-    else:
-        ALLOWED_CONTENT_TYPES = tuple(demisto.args()["allowedContentTypes"].split(","))  # type: ignore
-
-    # Work on the pcap file and return a result
-    http_flows = get_http_flows(pcap_file_path_in_container)
-
-    # Cut results according to the user args (times 2, because we are working on pairs of requests and responses).
-    if START:
-        http_flows = http_flows[int(START):]
-    if LIMIT:
-        http_flows = http_flows[:int(LIMIT)]
-
-    # Format and get output representation of the flows
-    formatted_http_flows = format_http_flows(http_flows, PYSHARK_RES_TO_DEMISTO, LIMIT_DATA, ALLOWED_CONTENT_TYPES)
-    markdown_output = get_markdown_output(formatted_http_flows)
-    context_output = formatted_http_flows
-
-    # Profit, send the output
-    demisto.results({"Type": entryTypes["note"],
-                     "ContentsFormat": formats["markdown"],
-                     "Contents": markdown_output,
-                     "EntryContext": {"PcapHTTPFlows": context_output}})
+        # Work on the pcap file and return a result
+        http_flows = get_http_flows(pcap_file_path_in_container)
+
+        # Cut results according to the user args (times 2, because we are working on pairs of requests and responses).
+        if START:
+            http_flows = http_flows[int(START):]
+        if LIMIT:
+            http_flows = http_flows[:int(LIMIT)]
+
+        # Format and get output representation of the flows
+        formatted_http_flows = format_http_flows(http_flows, PYSHARK_RES_TO_DEMISTO, LIMIT_DATA, ALLOWED_CONTENT_TYPES)
+        markdown_output = get_markdown_output(formatted_http_flows)
+        context_output = formatted_http_flows
+
+        # Profit, send the output
+        demisto.results({"Type": entryTypes["note"],
+                         "ContentsFormat": formats["markdown"],
+                         "Contents": markdown_output,
+                         "EntryContext": {"PcapHTTPFlows": context_output}})
+    except Exception as e:
+        sys.stderr = serr
```

Comment:
Don't we need to do that every time we call return_error?

Reply:
@yaakovi I don't know. but I think that this is not related to the changes, because until now we doesn't returned the exception in the return_error